src/map/entities/petentity.cpp

```diff
@@ -44,12 +44,13 @@ CPetEntity::CPetEntity(PET_TYPE petType)
 : CMobEntity()
 , m_PetType(petType)
 {
-    objtype        = TYPE_PET;
-    m_EcoSystem    = ECOSYSTEM::UNCLASSIFIED;
-    allegiance     = ALLEGIANCE_TYPE::PLAYER;
-    m_MobSkillList = 0;
-    m_PetID        = 0;
-    m_IsClaimable  = false;
+    objtype                 = TYPE_PET;
+    m_EcoSystem             = ECOSYSTEM::UNCLASSIFIED;
+    allegiance              = ALLEGIANCE_TYPE::PLAYER;
+    m_MobSkillList          = 0;
+    m_PetID                 = 0;
+    m_IsClaimable           = false;
+    m_bReleaseTargIDOnDeath = true;
```

Comment:
Can we move this down into battle entity now?

Reply:
the variable declaration itself? I don't see why not.